vehicle_profiles/hyundai/ioniq5-6.json

```diff
@@ -1,34 +1,134 @@
 {
-    "car_model": "Hyundai: Ioniq5/Ioniq6/Kona",
-    "init": "ATSH7E4;ATST96;",
-    "pids": [
-        {
-            "pid": "2201019",
-            "parameters": [
-                {
-                    "name": "SOC_BMS",
-                    "expression": "B10/2",
-                    "unit": "%",
-                    "class": "battery"
-                },
-                {
-                    "name": "Aux_Batt_Volts",
-                    "expression": "B38*0.1",
-                    "unit": "V",
-                    "class": "voltage"
-                }
-            ]
-        },
-        {
-            "pid": "2201057",
-            "parameters": [
-                {
-                    "name": "SOH",
-                    "expression": "[B34:B35]/10",
-                    "unit": "%",
-                    "class": "battery"
-                }
-            ]
+  "car_model": "Ioniq5/Ioniq6/Kona",
```

Comment:
Need to leave the name as-is, if the name changes it is a bit annoying atm in the wican GUI when updating, but also the Make prefix is there to make the list easier to find cars in, as more and more get added